### ISSUE TITLE
Shelly 2.5 automation

### ISSUE THREAD

**Issue by ravenjmsa** [StyraHem/ShellyForHASS#254]
Hello,
When trying to create an automation with shelly 2.5 roller using GUI I can't set the position and there's an error message when saving: "Message malformed: value is not allowed for dictionary value @ data['type']"
Thank you for your help

**Comment by chemelli74** [comment]
Please post automation code.

Simone

**Comment by ravenjmsa** [comment]
Hello,
I'm a first time user for hass I didn't learn how to code yet. As I wrote, I was using GUI and there was no change on automations.xml beacuse I coudn't save.
Best regards

**Comment by chemelli74** [comment]
> Hello,
> I'm a first time user for hass and I didn't learn how to code yet. As I wrote, I was using GUI and there was no change on automations.xml beacuse I coudn't save.
> Best regards

Please post a screenshot of the GUI then.

Simone

**Comment by ravenjmsa** [comment]
Here you go:

https://imgur.com/Vskk7yr

**Comment by hakana** [comment]
Yes, I don't know why, it worked in an earlier release of HA. I will check this.

**Comment by ravenjmsa** [comment]
Thank you

**Comment by SuperCocov9** [comment]
I just realize i've got the Same bug with Last beta1

**Comment by Cirrus0** [comment]
Can confirm the bug on my system as well, also i found the following bug that describes a similar issue with Ikea blinds:
https://community.home-assistant.io/t/creating-automation-that-triggers-ikea-blinds/204984
Might be connected.

**Comment by bqstony** [comment]
@ravenjmsa now in version 0.114.0 is it possible to set the position for automations in the GUI. No longer set it in yaml.

**Comment by lal12** [comment]
The issue seems to be that in the GUI the key positions is not set if the value is set to '0'. As a workaround you can create the automation with another value > 0 (e.g. 1) and afterwards edit it to 0.

I think this is an issue in HASS UI itself though.
